tests/integration/docker/volume.bats

```diff
@@ -48,6 +48,31 @@ setup() {
 	[ $RC -eq 0 ]
 }
 
+@test "Volume - bind-mount a directory" {
+	dir_path="$(pwd)/sharedSpace"
+	test_file="foo"
+	containerPath="/root/sharedSpace"
+	mkdir "$dir_path"
```

Comment:
what about this option:
```
mkdir "$dir_path"
touch "$dir_path/$test_file"
run $DOCKER_EXE run -i -v "$dir_path":"$containerPath" busybox ls "$containerPath" | grep "$test_file"
rm -r "$dir_path"
$DOCKER_EXE rm $($DOCKER_EXE ps -qa)
[ $status -eq 0 ]

```

I wouldn't like to manage the directory  under `setup()` and `teardown()` since it will be created and deleted for each test case we have in the `volume.bats` file